src/components/table/Table.vue

```diff
@@ -137,6 +137,12 @@
                                 </div>
                             </td>
                         </tr>
+                        <slot
```

Comment:
To improves the readability please, consider use the function with an if instead else-if